seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/multitablesink/MultiTableSinkWriter.java

```diff
@@ -243,4 +247,17 @@ public void close() throws IOException {
             throw new RuntimeException(firstE);
         }
     }
+
+    private void checkQueueRemain() {
```

Comment:
Also fix the bug when invoke `prepareCommit` and `abortPrepare` should make sure the SeaTunnelRow all push into sinkwriter.

Reply:
I worry if there exist some exception make the MultiTableWriterRunnable exist, then the blockingQueue will not empty, so the loop cannot exist.

Reply:
It will be checked in `subSinkErrorCheck` each loop. And let me add a test case to prove it.

Reply:
I added a test case for this. Please check again.